python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -677,6 +677,15 @@ router bgp {{ router_bgp.as }}
 {%             if peer_group.rcf_out is arista.avd.defined %}
       neighbor {{ peer_group.name }} rcf out {{ peer_group.rcf_out }}
 {%             endif %}
+{%             if peer_group.default_route.enabled is arista.avd.defined(true) %}
```

Comment:
Add similar template for address_family_evpn.neighbors

Reply:
Also if possible add all some example of neighbor, rcf and route map

Reply:
Done, please review